app/views/pages/2017/_ready_to_evolve.html.haml

```diff
@@ -65,7 +65,7 @@
                 freeze on her face. She lost her leg to a shark attack and her eye in a battle with
                 a french pirate.
           .five-columns
-            =image_tag('2017/pirata_rec.png', class: 'reylero-pirata')
+            =image_tag('2017/pirata_rec.svg', class: 'reylero-pirata')
```

Comment:
Missing space between *=* and *image_tag* I suggest = image_tag instead of =image_tag